tests/bootstrap.php

```diff
@@ -62,7 +65,15 @@
 
 global $CFG_GLPI, $GLPI_CACHE;
 
-include(GLPI_ROOT . "/inc/based_config.php");
+// check folder exists instead of class_exists('\GuzzleHttp\Client'), to prevent global includes
+if (file_exists($autoloadFile = __DIR__ . '/../vendor/autoload.php') && !file_exists(__DIR__ . '/../vendor/guzzlehttp/guzzle')) {
+    die("\nDevelopment dependencies not found\n\nrun: composer install -o\n\n");
+}
+
+require_once $autoloadFile;
```

Comment:
IMHO, it can be simplified.

```suggestion
require_once __DIR__ . '/../vendor/autoload.php';
```